frontend/src/pages/casedesigner/detail/BookcaseDetail.tsx

```diff
@@ -168,7 +167,13 @@ export default function BookcaseDetail() {
 
   const backButton = (
     <div className="flex col-4">
-      <BackButton className="ml-1" />
+      <Button
+        type="button"
```

Comment:
This is an unnecessary change if you update the id with useEffect. This also would break from the functionality that user expects of this button that is there for other pages.

Reply:
fixed